velox/functions/sparksql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -16,11 +16,24 @@
 
 #include "velox/common/base/tests/GTestUtils.h"
 #include "velox/functions/sparksql/tests/SparkFunctionBaseTest.h"
+#include "velox/type/Timestamp.h"
 #include "velox/type/tz/TimeZoneMap.h"
 
 namespace facebook::velox::functions::sparksql::test {
 namespace {
 
+std::string timestampToString(Timestamp ts) {
+  std::tm tm;
+  Timestamp::epochToUtc(ts.getSeconds(), tm);
```

Comment:
You're right. I think this might also not be used https://github.com/facebookincubator/velox/blob/83397d84f77a9abf9170a47d2a2ed266da20d54f/velox/type/tests/TimestampTest.cpp#L30

Reply:
I see. Maybe we can remove it in a timestamp relating PR later.